packages/lib-webpack/src/webpack/ValidateCompilationPlugin.ts

```diff
@@ -0,0 +1,29 @@
+import { WebpackPluginInstance, Compiler, WebpackError } from 'webpack';
+import { findPluginChunks } from '../utils/plugin-chunks';
+
+export class ValidateCompilationPlugin implements WebpackPluginInstance {
+  constructor(private readonly containerName: string, private readonly jsonpLibraryType: boolean) {}
+
+  apply(compiler: Compiler) {
+    compiler.hooks.afterCompile.tap(ValidateCompilationPlugin.name, (compilation) => {
+      const { runtimeChunk } = findPluginChunks(this.containerName, compilation);
+
+      if (runtimeChunk && this.jsonpLibraryType) {
```

Comment:
@vojtechszocs it would still be nice to link some sort of Webpack issue or doc link here since in theory this is a webpack issue - but I know we may not be able to find one.